bin/epr

```diff
@@ -144,11 +144,13 @@ def github(user, repo)
         item.number,
         item.user.login,
         item.title,
+        item.body[0..32700],
```

Comment:
What's this for?

Reply:
>  ... thought the problem was quotes in the body output, but it turned out to be a problem with Excel only permitting 32K characters in a cell.

What's your use case for having the full description? Maybe it's better to just always truncate at length N? Why not just use URL to view on web.

Reply:
I use the epr tool for writing release notes. It is helpful to have the description in the same view when writing them.  I could click on the link, but that would then switch from Excel to my browser and slow me down.  I’d be fine with limiting to a shorter description, perhaps 10000?

Reply:
Ok, makes sense. 

Looks like OpenOffice (well, LibreOffice) [has a 32,767 character limit too](https://libreofficehelp.com/maximum-number-rows-columns-cells-libreoffice-calc/). (Excel limits can be found [here](https://support.office.com/en-us/article/excel-specifications-and-limits-1672b34d-7043-467e-8e27-269d656771c3?ocmsassetID=HP010073849&CorrelationId=86614460-0060-4ca7-a4ed-ca6c8c3b21ee&ui=en-US&rs=en-US&ad=US#ID0EBABAAA=Newer_versions)). 

Given this, truncating at `2 ** 15 - 1` by default makes sense. But something alongs the lines of: 


```rb
# Top of epr code
DEFAULT_BODY_LENGTH = 2 * 32 - 1

# ...
# in existing method
rows << [
  "#{user}/#{repo}",
  item.pull_request ? TYPE_PR : TYPE_ISSUE,
  item.number,
  item.user.login,
  item.title,
  item.state,
  localtime(item.created_at),
  localtime(item.updated_at),
  item.html_url,
]

if $body
  body = item.body
  # -3 for "..."
  body = body.slice(0, DEFAULT_BODY_LENGTH - 3) << "..." if body.size > DEFAULT_BODY_LENGTH
  rows.insert_at(4, body)
end
```
And then I'll add an option later called: `--max-description=N`/`--no-max-description` that will allow one to adjust.